test/suites/node/reporting.js

```diff
@@ -9,25 +9,27 @@ describe("Reporting loop", function() {
         this.timeout(5000);
 
         var transport = new FileTransport(path.join(__dirname, '../../results/report_flush_loop.json'));
-        var runtime = Tracer.initNewTracer(LightStep.tracer({
-            access_token           : "{your_access_token}",
-            group_name             : "api-javascript/unit-test/report_flush_loop",
+        var tracer = LightStep.tracer({
```

Comment:
I want this to be `var tracer = OpenTracing.initNewTracer(LightStep.tracer({ ... }));`, but for reasons I don't understand it never flushes if I do that...